spec/after_commit_everywhere_spec.rb

```diff
@@ -20,8 +20,10 @@
   let(:handler) { spy("handler") }
 
   describe "#after_commit" do
+    let(:without_tx) { described_class::EXECUTE }
+
     subject do
-      example_class.new.after_commit do
+      example_class.new.after_commit(without_tx: without_tx) do
```

Comment:
I think that this test shouldn't be changed as it tests default behavior.
```suggestion
      example_class.new.after_commit do
```